cpu/stm32f4/include/periph_cpu.h

```diff
@@ -158,6 +158,28 @@ typedef struct {
 } dac_conf_t;
 
 /**
+ * @brief   Ethernet Peripheral configuration
+ */
+typedef struct {
+    enum {
+        MII = 18,                       /**< Configuration for MII */
+        RMII = 9,                       /**< Configuration for RMII */
+        SMI = 2,                        /**< Configuration for SMI */
+    } mode;                             /**< Select configuration mode */
+    enum {
+        HALFDUPLEX_10Mbps = 0x0000,
+        FULLDUPLEX_10Mbps = 0x0100,
+        HALFDUPLEX_100Mbps = 0x2000,
+        FULLDUPLEX_100Mbps = 0x2100,
+    } speed;                            /**< Speed selection */
+    char mac[6];                        /**< Ethernet MAC address */
+    char phy_addr;                      /**< PHY address */
+    gpio_t ports[];                     /**< Ports to use. MII requires 18 ports,
```

Comment:
I guess "ports" in this context means physical pins? (16 pins are grouped to a "port" in the STM32F407 CPU reference manual)